project_task_stage_change_restriction/models/project_task.py

```
@@ -0,0 +1,58 @@
+            if hasattr(task, 'user_ids') and user in task.user_ids:
+                return True
+
+        # "Project Manager Only": user must be the project manager
+        if new_stage.project_manager_only and user == task.project_id.user_id:
+            return True
+
+        # Group Members: user must belong to one of the allowed groups
+        if new_stage._user_passes_group_rule(user):
+            return True
+
+        return False
```

Ruff (0.11.9)

Return the condition `bool(new_stage._user_passes_group_rule(user))` directly

Replace with `return bool(new_stage._user_passes_group_rule(user))`

(SIM103)